spec/features/admin/enterprise_roles_spec.rb

```diff
@@ -133,6 +135,31 @@
           end
         end
       end
+
+      it "can invite unregistered users to be managers" do
+        new_email = 'new@manager.com'
+
+        find('a.button.help-modal').click
+        expect(page).to have_css '#invite-manager-modal'
```

Comment:
I find a bit weird to add assertions in the middle of the test's setup phase. As far as I understand we want to assert we reach a known state that confirms the user was invited, right?